src/styles/blocks/header.scss

```diff
@@ -0,0 +1,69 @@
+
+.upHeader {
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  margin-right: 50px;
+
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+    flex-direction: row;
+    text-transform: uppercase;
+  }
+
+  &__item {
+    display: flex;
+    flex-direction: column;
+    margin-left: 20px;
+    margin-bottom: 0;
+    flex-shrink: 0;
+  }
+
+  &__link {
+    display: flex;
+    flex-direction: column;
+    padding: 23px 0;
+    line-height: 14px;
+    text-decoration: none;
+    color: $link-color;
+  }
+
+  &__item:nth-child(1) {
+    margin-left: 0;
+  }
+
+  &__link:hover {
+    color: $hover-color;
+  }
+}
+
+.picLogo {
+  margin-left: 50px;
+}
+
+.picBox {
+  display: flex;
+  align-self: center;
+}
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 9th point

Reply:
I think they are already at this file - header.scss, its elements together here